<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomCtl</class>
 <widget class="QWidget" name="ZoomCtl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>60</width>
    <height>18</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>60</width>
    <height>18</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton#pBtn_Close{
	border-image: url(:/Assets/Image/red_nor.png);
}
QPushButton#pBtn_Close:hover{
	border-image: url(:/Assets/Image/red_on.png);
}

QPushButton#pBtn_Min{
	border-image: url(:/Assets/Image/yellow_nor.png);
}
QPushButton#pBtn_Min:hover{
	border-image: url(:/Assets/Image/yellow_on.png);
}

QPushButton#pBtn_Max{
	border-image: url(:/Assets/Image/green_nor.png);
}
QPushButton#pBtn_Max:hover{
	border-image: url(:/Assets/Image/green_on.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pBtn_Close">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pBtn_Min">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pBtn_Max">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
